code/modules/client/preferences.dm

```diff
@@ -113,9 +113,18 @@ GLOBAL_LIST_EMPTY(preferences_datums)
 		if(load_preferences() && load_character())
 			return
 
-	key_bindings = deepCopyList(GLOB.keybinding_list_by_key)
-
+	// We don't have a savefile or we failed to load them
 	random_character()
+	addtimer(CALLBACK(src, .proc/load_default_keybindings, C), 3 SECONDS)
+	
+
+/datum/preferences/proc/load_default_keybindings(client/C = usr)
+	to_chat(C, "Empty keybindings, setting defaults")
+
+	var/choice = tgalert(C, "Would you prefer 'hotkey' or 'classic' defaults?", "Setup keybindings", "hotkey", "classic")
+	hotkeys = (choice == "hotkey")
```

Comment:
delet this

Reply:
This was intentional because i want to set hotkey mode at the same time.